### PR TITLE
kiwi theme

### PR DESCRIPTION
**Pull request by Coding-Kiwi**
Hi
I started working on a dark theme materialize inspired theme.
it is far from being finished but it is perfectly usable as of now

### CODE REVIEW

**Comment by VeeK727** [comment]
This looks so good. I made one for myself and came here to create a pull request but this is better. Do fix the issues so it can be released.

File: kiwi/theme.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Dark",
```
Comment (mauriciofauth) [inline kiwi/theme.json:2]: I think this name is too generic. Maybe **Kiwi** or **Kiwi Dark**. What do you think?

Comment (Coding-Kiwi) [reply kiwi/theme.json:2]: Yes, sure! You can pick, I'm fine with both :D

**Comment by ibennetch** [comment]
I regret that I missed releasing this earlier this year when you submitted it. I'd love to publish it now. I see two small problems:

In the navigation area, I see this: 
image

If I test with the development release of 5.1 (manually changing the version in `theme.json`), I also have some trouble with the menu bar:
image

Any idea about either of these?

**Comment by Coding-Kiwi** [comment]
@ibennetch Some time has passed so the project wasn't on my radar ^^ I'll look into it the next days 👍

**Comment by ibennetch** [comment]
The menubar problem is related to phpmyadmin/phpmyadmin#15775, so that's easy to fix. I'm not as sure about what happened with the log in the navigation area.

**Comment by ibennetch** [comment]
The menubar problem is related to phpmyadmin/phpmyadmin#15775, so that's easy to fix.

I can release a 5.0 version of this and then fix the menu bar for 5.1 compatibility.

If you could look at the logo problem in the navigation area, that would be great. Thank you.

**Pull request closed without merging**